examples/postgres-RDS/README.md

```diff
@@ -0,0 +1,329 @@
+(1 row)
+
+## Add few more entries
+test=# INSERT INTO COMPANY (ID,NAME,AGE,ADDRESS,SALARY,CREATED_AT) VALUES (20, 'Omkar', 32, 'California', 20000.00, now());
+INSERT 0 1
+test=# INSERT INTO COMPANY (ID,NAME,AGE,ADDRESS,SALARY,CREATED_AT) VALUES (30, 'Prasad', 32, 'California', 20000.00, now());
+INSERT 0 1
+
+test=# select * from company;
+ id | name  | age |                      address                       | salary |         created_at
+----+-------+-----+----------------------------------------------------+--------+----------------------------
+ 10 | Paul  |  32 | California                                         |  20000 | 2019-09-16 14:39:36.316065
+ 20 | Omkar |  32 | California                                         |  20000 | 2019-09-16 14:40:52.952459
+ 30 | Omkar |  32 | California                                         |  20000 | 2019-09-16 14:41:06.433487
+```
+
+## Protect the Application
+
+You can now take a backup of the PostgresDB data using an ActionSet defining backup for this application. Create an ActionSet in the same namespace as the controller.
+
+```bash
+$ kubectl get profile -n postgres-test
+NAME               AGE
+s3-profile-7d6wt   7m25s
+
+$ kanctl create actionset --action backup --namespace kanister --blueprint rds-postgres-bp --statefulset postgres-test/my-release-postgresql --profile postgres-test/s3-profile-7d6wt
+actionset backup-llfb8 created
+
+$ kubectl --namespace kanister get actionsets.cr.kanister.io
+NAME           AGE
+backup-glptq   38s
+
+# View the status of the actionset
+$ kubectl --namespace kanister describe actionset backup-glptq
+```
+
+### Restore the Application to RDS Database instance
+
+To restore the data into RDS Postgres instance, you should use the backup that you created before. An easy way to do this is to leverage `kanctl`, a command-line tool that helps create ActionSets that depend on other ActionSets. Apart from this we also need to pass RDS instance details to the blueprint, the secret and configmap resource containing these details can be passed to kanctl:
+
+```bash
+$ kanctl --namespace kanister create actionset --action restore --from backup-glptq --config-maps dbconfig=postgres-test/dbconfig --secrets dbsecret=postgres-test/dbsecret
+actionset restore-backup-glptq-6jzt4 created
+
+## Check status
+$ kubectl --namespace kanister describe actionset restore-backup-glptq-6jzt4
+```
+
+Once the ActionSet status is set to "complete", you can see that the data has been successfully restored to the RDS PostgreSQL database instance
+
+To verify, Connect to the RDS PostgreSQL database instance 
+
+```bash
+$ kubectl run my-release-postgresql-client --rm --tty -i --restart='Never' --namespace postgres-test --image docker.io/bitnami/postgresql:15.1.0-debian-11-r31 --env="PGPASSWORD=<db-password>" --command -- psql --host <instance-name> -U <master-username> -d template1 -p 5432                   
```

Comment:
let's mention what should be the values of `instance-name`, and other placeholders.

Reply:
Do you mean using dummy values instead of placeholder? I modified these because of initial review comments I got [https://github.com/kanisterio/kanister/pull/1768#discussion_r1045580029](https://github.com/kanisterio/kanister/pull/1768#discussion_r1045580029) to replace values with placeholder.

Reply:
> Do you mean using dummy values instead of placeholder?


What I meant is, either before or after the `kubectl` command, we should mention the `instance-name`, `db-password` and `master-username` are the details of the RDS instance that we have already created as part of this step (link to the step where we create the RDS instance).

Reply:
I have addressed that. PTAL